packages/wonder-blocks-dropdown/components/single-select-menu.md

```diff
@@ -0,0 +1,114 @@
+### Single select menu. Regular and light versions.
+
+```js
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {StyleSheet} = require("aphrodite");
+
+const styles = StyleSheet.create({
+    row: {
+        flexDirection: "row",
+    },
+    sideMargins: {
+        marginRight: 10,
+    },
+    darkBackgroundWrapper: {
+        background: "black",
+        padding: 10,
+    },
+    strutLike: {
+        width: 8,
+    }
+});
+<View style={[styles.row]}>
+    <SingleSelectMenu
+        items={[
+            {
+                type: "select",
```

Comment:
This should always be `type: "select"` for `SingleSelectMenu` so we shouldn't need to pass this in.

Reply:
This is like this because I wasn't sure how to denote where a `SeparatorItem` would go without passing it in here (`type: "separator`), so I used [this pattern](https://flow.org/en/docs/types/unions/) to account for that.

Reply:
Actually, I'll leave the separator out of this one for now. I don't really see a use case where you would use a separator for a single select menu (will discuss with design).